benchmark/vm/run-in-context.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  n: [1],
```

Comment:
Yeah, I had to bump up `n` considerably to get the benchmarks to run long enough for me to be comfortable with the results, but then again I find myself having to do that a lot with many of the other existing benchmarks ...